forge-gui/res/cardsfolder/upcoming/oskar_rubbish_reclaimer.txt

```diff
@@ -0,0 +1,10 @@
+Name:Oskar, Rubbish Reclaimer
+ManaCost:3 U B
+Types:Legendary Creature Human Wizard
+PT:3/3
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {1} less to cast for each card type among cards in your graveyard.
+SVar:X:Count$ValidGraveyard Card.YouOwn$DifferentCMC
+T:Mode$ Discarded | ValidPlayer$ You | ValidCard$ Card.nonLand | TriggerZones$ Battlefield | Execute$ TrigPlay | TriggerDescription$ Whenever you discard one or more nonland cards, you may exile one of them from your graveyard. If you do, you may cast it this turn.

```

Comment:
wrong trigger
this says "one or more" = DiscardedAll

Reply:
Noticed the trigger but missed the static! Thanks!